Fritz!DECT200/210 Wall Plugs shown unknown after OH 4.0.0M3 update

<!-- Provide a general summary of the issue in the *Title* above --> All DECT Devices where working well in OH 3.4.4. After Upgrade to OH 4.0.0M3 some of the DECT 200 / DECT 210 Wallplugs are shown "Unknown" instead of Online. See [https://community.openhab.org/t/avm-dect-200-wallplugs-not-working-in-oh4-0-0-m3/147512/28](https://community.openhab.org/t/avm-dect-200-wallplugs-not-working-in-oh4-0-0-m3/147512/28) for details
<!-- If the issue is related to a binding, please include its short name in --> AVM Fritz!Box Binding
<!-- square brackets in the title - Example: "[astro] My issue..." -->

Screenshots shows one "Online" and the other wallplug "Unknown":
![image](https://github.com/openhab/openhab-addons/assets/25499089/2f5df894-9fc6-45ca-9a95-a317477f02f5)

In summary 4 of 7 are unknown.
I tried removing and adding again the Devices, Removing and adding again the Fritzbox and new installation of binding.
It always results in 4 unknown and 3 online Fritz DECT 200 Things.

I made the update twice on 2 different systems. Always the same.

If you need further informations, you can contact me anytime.
<!-- Important: Please contact the openHAB community forum for questions or -->
<!-- for configuration and usage guidance: https://community.openhab.org -->

<!-- Feel free to delete any comment lines in the template (starting with "<!--") -->

## Expected Behavior
<!-- If you're describing a bug, tell us what should happen -->
<!-- If you're suggesting a change/improvement, tell us how it should work -->
All DECT200/210 Devices should be online, as in OH 3.4.4 version
## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior -->
<!-- Include related log information (preferably debug level) and related configs -->
<!-- Use a file attachment for log and config information longer than a few lines -->
<!-- Enclose multi-line log/code snippets with ``` on new lines for proper formatting -->
<!-- If suggesting a change/improvement, explain the difference from current behavior -->
<!-- For improvements, discuss at community.openhab.org first and include link to topic -->

## Possible Solution
<!-- Not obligatory, but suggest a fix/reason for the bug, -->
<!-- or ideas how to implement the addition or change -->

## Steps to Reproduce (for Bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps to -->
<!-- reproduce this bug. Include code to reproduce, if relevant -->
1. Update from OH 3.4.4 to 4.0.0M3
2. Check DECT200/210 in MainUI

## Context
<!-- How has this issue affected you? What are you trying to accomplish? -->Update to OH 4.0.0 leads to non functional AVM Fritz!DECT 200 and 210 Devices
<!-- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version used:Openhab 4.0.0M3
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...): see below
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...):
```
openhabian@oh40test:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

Linux oh40test 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux

openjdk 17.0.6 2023-01-17
OpenJDK Runtime Environment (build 17.0.6+10-Raspbian-1deb11u1rpt1)
OpenJDK Client VM (build 17.0.6+10-Raspbian-1deb11u1rpt1, mixed mode, emulated-client)
```

Again updated from (working) OH 3.4.4 to 4.0.0M3. Again 4 of 7 DECT 200/210 Wallplugs are unknown after upgrade.
Here's the openhablog.log after restart:
`
2023-06-30 09:58:10.636 [WARN ] [core.thing.internal.ThingManagerImpl] - Channel types or config descriptions for thing 'avmfritz:FRITZ_DECT_210:192_168_1_1:116570065738' are missing in the respective registry for more than 120s. In case it does not happen immediately after an upgrade, it should be fixed in the binding.
2023-06-30 09:58:10.639 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_210:192_168_1_1:116570065738:power' because channel type 'system:electrical-power' could not be found.
2023-06-30 09:58:10.641 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_210:192_168_1_1:116570065738:voltage' because channel type 'system:electrical-voltage' could not be found.
2023-06-30 09:58:10.643 [INFO ] [core.thing.internal.ThingManagerImpl] - Updating 'avmfritz:FRITZ_DECT_210:192_168_1_1:116570065738' from version 0 to 1
2023-06-30 09:58:10.851 [WARN ] [core.thing.internal.ThingManagerImpl] - Channel types or config descriptions for thing 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610004856' are missing in the respective registry for more than 120s. In case it does not happen immediately after an upgrade, it should be fixed in the binding.
2023-06-30 09:58:10.856 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610004856:power' because channel type 'system:electrical-power' could not be found.
2023-06-30 09:58:10.858 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610004856:voltage' because channel type 'system:electrical-voltage' could not be found.
2023-06-30 09:58:10.860 [INFO ] [core.thing.internal.ThingManagerImpl] - Updating 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610004856' from version 0 to 1
2023-06-30 09:58:10.889 [WARN ] [core.thing.internal.ThingManagerImpl] - Channel types or config descriptions for thing 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610328309' are missing in the respective registry for more than 120s. In case it does not happen immediately after an upgrade, it should be fixed in the binding.
2023-06-30 09:58:10.893 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610328309:power' because channel type 'system:electrical-power' could not be found.
2023-06-30 09:58:10.896 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610328309:voltage' because channel type 'system:electrical-voltage' could not be found.
2023-06-30 09:58:10.898 [INFO ] [core.thing.internal.ThingManagerImpl] - Updating 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610328309' from version 0 to 1
2023-06-30 09:58:10.931 [WARN ] [core.thing.internal.ThingManagerImpl] - Channel types or config descriptions for thing 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296701' are missing in the respective registry for more than 120s. In case it does not happen immediately after an upgrade, it should be fixed in the binding.
2023-06-30 09:58:10.936 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296701:power' because channel type 'system:electrical-power' could not be found.
2023-06-30 09:58:10.938 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296701:voltage' because channel type 'system:electrical-voltage' could not be found.
2023-06-30 09:58:10.941 [INFO ] [core.thing.internal.ThingManagerImpl] - Updating 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296701' from version 0 to 1
2023-06-30 09:58:11.118 [WARN ] [core.thing.internal.ThingManagerImpl] - Channel types or config descriptions for thing 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296702' are missing in the respective registry for more than 120s. In case it does not happen immediately after an upgrade, it should be fixed in the binding.
2023-06-30 09:58:11.125 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296702:power' because channel type 'system:electrical-power' could not be found.
2023-06-30 09:58:11.130 [ERROR] [.update.UpdateChannelInstructionImpl] - Failed to create channel 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296702:voltage' because channel type 'system:electrical-voltage' could not be found.
2023-06-30 09:58:11.134 [INFO ] [core.thing.internal.ThingManagerImpl] - Updating 'avmfritz:FRITZ_DECT_200:192_168_1_1:087610296702' from version 0 to 1
`

--- a/bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml
+++ b/bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml
@@ -241,7 +241,7 @@
 			<channel id="locked" typeId="locked"/>
 			<channel id="device_locked" typeId="device_locked"/>
 			<channel id="temperature" typeId="temperature"/>
-			<channel id="energy" typeId="system.electrical-energy">
+			<channel id="energy" typeId="system.electric-energy">
 				<label>Energy Consumption</label>
 				<description>Accumulated energy consumption.</description>
 			</channel>
@@ -275,7 +275,7 @@
 			<channel id="locked" typeId="locked"/>
 			<channel id="device_locked" typeId="device_locked"/>
 			<channel id="temperature" typeId="temperature"/>
-			<channel id="energy" typeId="system.electrical-energy">
+			<channel id="energy" typeId="system.electric-energy">
 				<label>Energy Consumption</label>
 				<description>Accumulated energy consumption.</description>
 			</channel>
@@ -309,7 +309,7 @@
 			<channel id="mode" typeId="mode"/>
 			<channel id="locked" typeId="locked"/>
 			<channel id="device_locked" typeId="device_locked"/>
-			<channel id="energy" typeId="system.electrical-energy">
+			<channel id="energy" typeId="system.electric-energy">
 				<label>Energy Consumption</label>
 				<description>Accumulated energy consumption.</description>
 			</channel>
@@ -515,7 +515,7 @@
 			<channel id="mode" typeId="mode"/>
 			<channel id="locked" typeId="locked"/>
 			<channel id="device_locked" typeId="device_locked"/>
-			<channel id="energy" typeId="system.electrical-energy">
+			<channel id="energy" typeId="system.electric-energy">
 				<label>Energy Consumption</label>
 				<description>Accumulated energy consumption.</description>
 			</channel>

--- a/bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/update/instructions.xml
+++ b/bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/update/instructions.xml
@@ -15,6 +15,17 @@
 				<type>system:electrical-voltage</type>
 			</update-channel>
 		</instruction-set>
+		<instruction-set targetVersion="2">
+			<update-channel id="energy">
+				<type>system:electric-energy</type>
+			</update-channel>
+			<update-channel id="power">
+				<type>system:electric-power</type>
+			</update-channel>
+			<update-channel id="voltage">
+				<type>system:electric-voltage</type>
+			</update-channel>
+		</instruction-set>
 	</thing-type>
 
 	<thing-type uid="avmfritz:FRITZ_DECT_210">
@@ -27,6 +38,17 @@
 			</update-channel>
 			<update-channel id="voltage">
 				<type>system:electrical-voltage</type>
+			</update-channel>
+		</instruction-set>
+		<instruction-set targetVersion="2">
+			<update-channel id="energy">
+				<type>system:electric-energy</type>
+			</update-channel>
+			<update-channel id="power">
+				<type>system:electric-power</type>
+			</update-channel>
+			<update-channel id="voltage">
+				<type>system:electric-voltage</type>
 			</update-channel>
 		</instruction-set>
 	</thing-type>
@@ -43,6 +65,17 @@
 				<type>system:electrical-voltage</type>
 			</update-channel>
 		</instruction-set>
+		<instruction-set targetVersion="2">
+			<update-channel id="energy">
+				<type>system:electric-energy</type>
+			</update-channel>
+			<update-channel id="power">
+				<type>system:electric-power</type>
+			</update-channel>
+			<update-channel id="voltage">
+				<type>system:electric-voltage</type>
+			</update-channel>
+		</instruction-set>
 	</thing-type>
 
 	<thing-type uid="avmfritz:FRITZ_Powerline_546E">
@@ -55,6 +88,17 @@
 			</update-channel>
 			<update-channel id="voltage">
 				<type>system:electrical-voltage</type>
+			</update-channel>
+		</instruction-set>
+		<instruction-set targetVersion="2">
+			<update-channel id="energy">
+				<type>system:electric-energy</type>
+			</update-channel>
+			<update-channel id="power">
+				<type>system:electric-power</type>
+			</update-channel>
+			<update-channel id="voltage">
+				<type>system:electric-voltage</type>
 			</update-channel>
 		</instruction-set>
 	</thing-type>

